.azure-pipelines/client.yml

```diff
@@ -22,6 +22,10 @@ jobs:
           versionSpec: '$(NodeVersion)'
         displayName: 'Install Node.js $(NodeVersion)'
 
+      - script: |
```

Comment:
I don't think we want to blindly install latest. We should specify the version we want to use as part of the install command.